ee_tests/Dockerfile.builder

```diff
@@ -64,11 +64,13 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
 # && yum -y --setopt tsflags='nodocs' install ffmpeg
 
 # Provide oc client to tests Clean up the test user account's resources in OpenShift Online
-RUN wget https://github.com/openshift/origin/releases/download/v1.5.0/openshift-origin-client-tools-v1.5.0-031cbe4-linux-64bit.tar.gz &&\
-    tar -xzvf openshift-origin-client-tools-v1.5.0-031cbe4-linux-64bit.tar.gz &&\
-    mv openshift-origin-client-tools-v1.5.0-031cbe4-linux-64bit/oc oc
+RUN wget https://mirror.openshift.com/pub/openshift-v3/clients/3.10.0-0.50.0/linux/oc.tar.gz &&\
+    tar -xzvf oc.tar.gz && mv oc /usr/bin/oc
 
-# install all dependencies
+# Install jq
+RUN yum install --setopt tsflags='nodocs' -y jq
```

Comment:
why not to install it on line 33?

Reply:
Because the comment above line 33 says 'install chrome' and this is not required by chrome. I am worried that we would loose track of what is installed and why.

Reply:
the comment is misleading, please move `jq` somewhere to the `yum install` starting on line 34 (definitely not on line 39) and remove/update the comment